src/test/java/io/vertx/httpproxy/ProxyRequestTest.java

```diff
@@ -79,6 +78,46 @@ private void testChunkedBackendResponse(TestContext ctx, HttpVersion version) {
     HttpClient httpClient = vertx.createHttpClient(new HttpClientOptions().setProtocolVersion(version));
     httpClient.request(HttpMethod.GET, 8080, "localhost", "/somepath")
         .compose(req -> req.send().compose(HttpClientResponse::body))
+        .onComplete(result -> {
+          if (version == HttpVersion.HTTP_1_0) {
```

Comment:
`ctx.assertEquals(result.succeeded(), version !=HttpVersion_HTTP_1_0)`

Reply:
From what I understand HTTP/2 doesn't support chunked anymore (in favor of other built-in streaming methods). I changed it to succeed only for HTTP/1.1.